nextflow_schema.json

```diff
@@ -574,19 +574,156 @@
             "description": "Options to related to metagenomic screening.",
             "default": "",
             "properties": {
-                "run_metagenomicscreening": {
+                "run_metagenomics_screening": {
                     "type": "boolean",
                     "description": "Turn on metagenomic screening of mapped, unmapped, or all reads.",
                     "fa_icon": "fas fa-power-off",
                     "help_text": "Turns on the metagenomic screening subworkflow of the pipeline, where reads are screened against large databases. Typically used for pathogen screening or microbial community analysis.\n\nIf supplied, this will also turn on the BAM filtering subworkflow of the pipeline."
                 },
-                "metagenomicscreening_input": {
+                "metagenomics_screening_input": {
                     "type": "string",
                     "default": "unmapped",
                     "description": "Specify which type of reads to go into metagenomic screening.",
                     "enum": ["unmapped", "mapped", "all"],
                     "fa_icon": "fas fa-hand-pointer",
                     "help_text": "You can select which reads coming out of the read alignment step will be sent for metagenomic analysis.\n\nThis influences which reads are sent to this step, whether you want unmapped reads (used in most cases, as 'host reads' can often be contaminants in microbial genomes), mapped reads (e.g, when doing competitive against a genomic reference of multiple genomes and which to apply LCA correction), or all reads.\n\n> \u26a0\ufe0f If you skip paired-end merging, all reads will be screened as independent reads - not as pairs! - as all FASTQ files from BAM filtering are merged into one. This merged file is _not_ saved in results directory.\n\n> Modifies tool parameter(s):\n> - samtools fastq: `-f 4` / `-F 4`"
+                },
+                "metagenomics_profiling_tool": {
+                    "type": "string",
+                    "description": "Specify which tool to use for metagenomic profiling and screening.",
+                    "enum": ["malt", "metaphlan2", "kraken2", "krakenuniq"],
+                    "fa_icon": "fas fa-toolbox",
+                    "help_text": "Select which tool to run metagenomics profiling on designated metagenomics_screening_input. These tools behave vastly differently due to performing read profiling using different methods and yield vastly different reuslts."
+                },
+                "metagenomics_profiling_database": {
+                    "type": "string",
+                    "format": "directory-path",
+                    "description": "Specify a databse directory to run metagenomics profiling on. In the case of kraken2, this can be a tar.gz of the directory.",
```

Comment:
```suggestion
                    "description": "Specify the path to a database directory to run metagenomics profiling on. In the case of kraken2, this can be a tar.gz of the directory.",
```